src/components/Todo/Todo.tsx

```diff
@@ -36,17 +37,24 @@ export interface TodoAppProps {
   defaultItems?: TodoItem[];
   onChange: (items: TodoItem[]) => void;
 }
-
 function TodoApp(props: TodoAppProps) {
   const { defaultItems = [], onChange } = props;
   const [items, setItems] = useState<TodoItem[]>(defaultItems);
+  const [undo, setUndo] = useState<TodoItem[]>(defaultItems);
```

Comment:
this should be an array right? 
In this implementation undo will only work for single undo but multiple undo should also work.